### DISCUSSION TITLE
Idea: is it possible to make a simple web server in koka now ?

### DISCUSSION THREAD

**Discussion by lengyijun** [koka-lang/koka#474] [General]
I want to do some practise in koka, maybe a webserver is a humble start ?

**Comment by TimWhiting** [comment]
It might be best to wait for async support before doing a webserver. I have a PR open for it right now: https://github.com/koka-lang/koka/pull/384, so if you want I can create a release bundle that supports async / and has async networking apis, and drop the files here, I would just need to know if I need to create it for windows/macos/linux.

As a humble start, it seems most people try creating their own parsing libraries, but do whatever most interests you!
